src/helpers/AvatarHelper.ts

```diff
@@ -43,6 +43,19 @@ export function getInitials(name: string) {
   return _.toUpper(initials);
 }
 
+export function getBackgroundColor(name?: string,
```

Comment:
consider adding few simple tests to cover the new functionalities

Reply:
Done (found a bug!)

Reply:
The power of testings (: